test/src/analyzers/lint_analyzer/rules/rules_list/avoid_global_state/examples/example.dart

```diff
@@ -0,0 +1,6 @@
+var answer = 42; // LINT
+var evenNumbers = [1, 2, 3].where((element) => element.isEven); // LINT
+
+class Foo {
+  static int? bar;
+}
```

Comment:
I'd suggest more code here just to be sure everything is okay. Like some methods, fields, variable declarations, etc.